snapcraft/plugins/copy.py

```diff
@@ -57,6 +64,6 @@ def build(self):
             os.makedirs(os.path.dirname(dst), exist_ok=True)
 
             if os.path.isdir(src):
-                shutil.copytree(src, dst)
+                shutil.copytree(src, dst, symlinks=self.options.follow_symlinks)
```

Comment:
I'm +1 on doing sanity checks for the resulting snap and/or preventing it from generating stupid things.